src/style.css

```diff
@@ -1 +1,67 @@
-/* add styles here */
+:root {
+  --main-color: #e1e7ed;
+  --text-color: #3d4e61;
+  --shadow: rgba(0, 0, 0, 0.25);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  width: 100%;
+  height: 70px;
+  color: var(--text-color);
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 19px;
+  border: 1px solid var(--main-color);
+  border-radius: 4px;
+
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: left 25px top 24px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.search-bar__input--small {
+  background-size: 11px;
+  background-position: left 12px bottom 15px;
+  height: 42px;
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.search-bar__input::placeholder {
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you could specify a generic font family like `sans-serif` as a fallback after your custom font.